### PR TITLE
Add fix connected with wrong data type in in_array function.

### PR DESCRIPTION
**Pull request by coditive-rafal-o**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
 # Shipping via Planzer for WooCommerce
+
 Tags: planzer, shipping, e-commerce, store, sales, sell, woo, shop, cart, checkout, woo commerce
-Tested up to: 6.3 Stable
-tag: 1.0.15
+Tested up to: 6.1 Stable
```
Comment (noahmenzi) [inline README.md:4]: This is not changed by purpose, right?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+
 Shipping via Planzer for WooCommerce automatically franks your orders thanks to the connection to Planzer. Besides, it directly generates a delivery bill or label. Save time and focus on growing your store.
 
 ### Advantages at a glance
 
-* **Easy installation**<br />The plugin can be easily installed and filled with your data. Once the plugin is activated and in live mode, Planzer will pick up the packages from you.
-* **More time for the important things**<br />With our plugin, you no longer have to manually enter orders in the Planzer portal. This saves you time and allows you to take care of your daily business.
-* **Lower error rate**<br /> Since the orders are automatically transferred according to the input of your customers, (including shipping and billing addresses) the error rate will automatically decrease.
-* **QR code with personalized delivery note**<br /> On request, a personalized delivery bill is generated instead of the simple label with QR code.
+- **Easy installation**<br />The plugin can be easily installed and filled with your data. Once the plugin is activated and in live mode, Planzer will pick up the packages from you.
```
Comment (noahmenzi) [inline README.md:15]: Is it correct that all * are replaced by "-"?

File: readme.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 === Shipping via Planzer for WooCommerce ===
 Tags: planzer, shipping, e-commerce, store, sales, sell, woo, shop, cart, checkout, woo commerce
-Tested up to: 6.3
-Stable tag: 1.0.15
+Tested up to: 6.1
```
Comment (noahmenzi) [inline readme.txt:3]: same here.

**Review by noahmenzi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (coditive-rafal-o) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (coditive-rafal-o) [reply README.md]: Fixed

File: readme.txt (hunk 3 above)
Comment (coditive-rafal-o) [reply readme.txt]: Fixed

**Comment by noahmenzi** [comment]
@coditive-rafal-o My comments are not fixed yet, right? Thanks for checking again.

**Review by noahmenzi — CHANGES_REQUESTED** [review]

**Review by noahmenzi — APPROVED** [review]
Cool, now everything looks fine, thanks!

**Pull request merged**